Makefile

```diff
@@ -1,6 +1,6 @@
 BIN_NAME=terraform-provider-instaclustr
 
-VERSION=2.0.9
+VERSION=2.0.10
```

Comment:
I am making this 2.0.10 in https://github.com/instaclustr/terraform-provider-instaclustr/pull/219/files to test the deployment pipeline. 
Do you want to change this or should we be taking this together?